codebundles/aws-c7n-ebs-health/runbook.robot

```diff
@@ -0,0 +1,155 @@
+*** Settings ***
+Metadata          Author    saurabh3460
+Metadata          Supports    AWS    EBS    CloudCustodian
+Metadata          Display Name    AWS EBS Health
+Documentation     Check for AWS EBS resources by identifying unattached volumes, unused snapshots, and unencrypted volumes.
+Force Tags    EBS    Volume    AWS    Storage    Encryption
+
+Library    RW.Core
+Library    RW.CLI
+Library    CloudCustodian.Core
+
+Suite Setup    Suite Initialization
+
+
+
+*** Tasks ***
+List Unattached EBS Volumes in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}` 
+    [Documentation]  Check for unattached EBS volumes in the specified region. 
+    [Tags]    ebs    storage    aws    volume    unattached
+    ${c7n_output}=    RW.CLI.Run Cli
+    ...    cmd=custodian run -r ${AWS_REGION} --output-dir ${OUTPUT_DIR}/aws-c7n-ebs-health ${CURDIR}/unattached-ebs-volumes.yaml --cache-period 0
+    ...    secret__aws_account_id=${AWS_ACCESS_KEY_ID}
+    ...    secret__aws_secret_access_key=${AWS_SECRET_ACCESS_KEY}
+    ${report_data}=     RW.CLI.Run Cli
+    ...    cmd=cat ${OUTPUT_DIR}/aws-c7n-ebs-health/unattached-ebs-volumes/resources.json 
+    RW.Core.Add Pre To Report    ${c7n_output.stdout}     # Data needs to be parsed to be usable in the report.
+
+    ${parsed_results}=    CloudCustodian.Core.Parse EBS Results
+    ...    input_dir=${OUTPUT_DIR}/aws-c7n-ebs-health
+    RW.Core.Add Pre To Report    ${parsed_results} 
+
+    ${clean_output_dir}=    RW.CLI.Run Cli    cmd=rm -rf ${OUTPUT_DIR}/aws-c7n-ebs-health/unattached-ebs-volumes
+    # Convert custodian json output to a list.
+    TRY
+        ${ebs_volume_list}=    Evaluate    json.loads(r'''${report_data.stdout}''')    json
+    EXCEPT
+        Log    Failed to load JSON payload, defaulting to empty list.    WARN
+        ${ebs_volume_list}=    Create List
+    END
+
+    IF    len(@{ebs_volume_list}) > 0
+        FOR    ${item}    IN    @{ebs_volume_list}
+            RW.Core.Add Issue        
+            ...    severity=4
+            ...    expected=EBS volumes in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}` should be attached.
+            ...    actual=EBS volume `${item["VolumeId"]}` in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}` is unattached.
+            ...    title=Unattached EBS volume `${item["VolumeId"]}` detected in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`.
+            ...    reproduce_hint=`${c7n_output.cmd}`
+            ...    details=${item}
+            ...    next_steps=Escalate to service owner to review of unattached AWS EBS volume in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`.\nDelete unattached AWS EBS volume in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`.
```

Comment:
@saurabh3460 Please change this text to EXACTLY the following and apply a similar pattern for the rest of the next steps in other tasks. 

            ...    next_steps=Delete Unattached AWS EBS volumes in AWS Region \`${AWS_REGION}\` in AWS account \`${AWS_ACCOUNT_ID}\`\nEscalate Unattached AWS EBS volumes in AWS Region \`${AWS_REGION}\` in AWS account \`${AWS_ACCOUNT_ID}\` for review
            
            
![image](https://github.com/user-attachments/assets/233e7a01-8dd9-4090-9afc-b0ba12a4ce2b)

Reply:
It works, Thanks!!